### DISCUSSION TITLE
GitHub Workflow Action Badge

### DISCUSSION THREAD

**Discussion by CalvinAllen** [badges/shields#7247] [Q&A] [answered]
I'm having problems getting this badge to work, and before I open an issue, just wanted to see if I'm doing something incorrectly somewhere along the way.

Repo/Action: 
https://github.com/CalvinAllen/OpenInNotepadPlusPlus/actions/workflows/release_build_and_deploy.yml

Based on that information, I configured:
https://img.shields.io/github/workflow/status/CalvinAllen/OpenInNotepadPlusPlus/release_build_and_deploy?logo=github&style=for-the-badge

Unfortunately, I simply get an error badge of "repo, branch, or workflow not found"
![](https://img.shields.io/github/workflow/status/CalvinAllen/OpenInNotepadPlusPlus/release_build_and_deploy?logo=github&style=for-the-badge)

I installed the application to get around throttle limits, thinking it may have been related, but that didn't help either.

Any thoughts?

**Comment by chris48s** [comment, marked as answer]
The github workflow badge uses the action name not the filename, so the correct url for this should be
https://img.shields.io/github/workflow/status/CalvinAllen/OpenInNotepadPlusPlus/Open%20In%20Notepad%2B%2B%20-%20VS%20Legacy?logo=github&style=for-the-badge

**Reply by CalvinAllen** [reply]
I wondered if that was the case, think that was the only thing I didn't try.  Thanks!

**Reply by dxvv-software** [reply]
k https://img.shields.io/github/workflow/status/iDevYT/freeshell/pages-build-deployment?label=Pages%20Build what is mine
